docs/permissions-and-roles.md

```diff
@@ -0,0 +1,31 @@
+# Permissions and roles
+
+'mediator':
```

Comment:
```suggestion
## `mediator`:
```

Just a suggestion to use headlines as this seems to be what it is. Take it or leave it 😄